FE/todo-app/src/components/SideBar.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ReactNode } from "react";
+import { BsPlus, BsCheckLg, BsCollection } from "react-icons/bs";
+
+export default function SideBar() {
+  return (
+    <div className="fixed top-0 left-0 h-screen w-16 mt-0 flex flex-col bg-myPrimary text-white shadow-lg">
+      <SideBarIcon icon={<BsPlus size="28" />} text="Add New Group" />
+      <SideBarIcon icon={<BsCheckLg size="28" />} text="Completed" />
+      <SideBarIcon icon={<BsCollection size="28" />} text="All" />
+    </div>
+  );
+}
+
+type SideBarIconProps = {
+  icon: ReactNode;
+  text?: string;
+};
+
+const SideBarIcon = ({ icon, text }: SideBarIconProps) => (
```

Comment:
vím že ti v poště říkaly aby jsi to dělal takhle, ale prosím nedělej to, díky tomuto tě TS nehlídá protože to bere jako funkci a vůbec neví že jde o reactí komponentu.. prosím o 
`const SideBarIcon: FC<Props> = ({}) => {...}`
Propsy vždy stačí pojmenovat jen jako Props, prootže je neexportuješ.